### ISSUE TITLE
Getting JSONException when using --get-O365-drive-id command

### ISSUE THREAD

**Issue by mbkupfer** [abraunegg/onedrive#289]
### Bug Report Details ###
**Describe the bug**
I was following the instruction in README.Office365.md, but couldn't get the id for a shared folder. I tried all 3 of my shared folders and none of them worked. Each gave me an error. 

Getting JSONException when using ```--get-O365-drive-id ```

**Application and Operating System Details:**
- OS: Output of `uname -a` & provide your OS & version (CentOS 6.x, Ubuntu 18.x etc)
Ubuntu 18
- Are you using a headless system (no gui) or with a gui installed?
No
- Application version: Output of `onedrive --version`
onedrive v2.2.1-7-g91aaf91

- OneDrive Account Type
Business 

- DMD or LDC compiler version `dmd --version` or `ldmd2 --version`
dmd-2.083.1

**Complete Verbose Log Output**
A clear and full log of the problem when running the application in the following manner (ie, not in monitor mode):
```
mkupfer@ubuntu:~$ onedrive --get-O365-drive-id Marketing
Initializing the Synchronization Engine ...
std.json.JSONException@std/json.d(540): Key not found: description
----------------
/home/mkupfer/dlang/dmd-2.083.1/linux/bin64/../../src/phobos/std/exception.d:515 pure @safe void std.exception.bailOut!(std.json.JSONException).bailOut(immutable(char)[], ulong, scope const(char)[]) [0x4839b463]
??:? pure @safe inout(std.json.JSONValue)* std.exception.enforce!(std.json.JSONException).enforce!(inout(std.json.JSONValue)*).enforce(inout(std.json.JSONValue)*, lazy const(char)[], immutable(char)[], ulong) [0x48444696]
??:? inout pure ref @safe inout(std.json.JSONValue) std.json.JSONValue.opIndex(immutable(char)[]) [0x483f8659]
src/sync.d:1764 void sync.SyncEngine.querySiteCollectionForDriveID(immutable(char)[]) [0x483d022c]
src/main.d:320 _Dmain [0x483a9cba]

```

**Screenshots**

Here is a snippet from my onedrive showing that this folder does indeed exist
![screenshot at 2018-12-17 16-12-51](https://user-images.githubusercontent.com/6313008/50123796-000f0b00-0217-11e9-94d9-22b8f353e4da.png)

### Bug Report Checklist ###
- [x] Detailed description
- [x] Reproduction steps (if applicable)
- [x] Verbose Log Output

**Comment by abraunegg** [comment]
@mbkupfer
Can you run the following:
```
```

**Comment by mbkupfer** [comment]
Some of the stuff looks in the response looks like token and some have other things that look questionably private. Any specific line you are looking for, or can I send you an email?

**Comment by abraunegg** [comment]
@mbkupfer 
The line I am after will be right before the crash which has the JSON response - for example:
```
OneDrive API Response: {"@odata.context":"https:\/\/graph.microsoft.com\/v1.0\/$metadata#drives","value":[{"createdBy":{"user":{"displayName":"System Account"}},"createdDateTime":"2018-11-24T09:27:43Z","description":"","driveType":"documentLibrary","id":"b!6H_y8BWP8Uqg2X0QAy4XfxiTMFe_reZKrMZXLeBjL4g70-n16rIGQpOG_l7vVxU5","lastModifiedDateTime":"2018-12-03T07:20:11Z","name":"Documents","owner":{"group":{"displayName":"ab-github-odtest Owners","email":"ab-github-odtest@mynasau.onmicrosoft.com","id":"3b5679dd-c264-4ad2-97d5-f29ff3106307"}},"quota":{"deleted":691,"remaining":27487788979967,"state":"normal","total":27487790694400,"used":30804},"webUrl":"https:\/\/mynasau.sharepoint.com\/sites\/ab-github-odtest\/Shared%20Documents"}]}
```

You can email that response to support@mynas.com.au if you feel too sensitive.

**Comment by mbkupfer** [comment]
I will shortly send to the email you have given. Thanks.

**Comment by abraunegg** [comment]
@mbkupfer 
Thanks for that. Issue found, will post a patch shortly

**Comment by mbkupfer** [comment]
Very nice! It is now working for me. 

Thank you for responding to this issue so quickly, and thank you for creating such a useful library for linux users.

**Comment by abraunegg** [comment]
Resolved by PR #290
